_tools/check_docs.ts

```diff
@@ -67,7 +72,8 @@ const ENTRY_POINTS = [
 ] as const;
 
 const TS_SNIPPET = /```ts[\s\S]*?```/g;
-const ASSERTION_IMPORT = /import \{.*\} from "@std\/assert(?:\/.*)?";/gm;
+const ASSERTION_IMPORT =
+  /from "@std\/(assert(\/[a-z-]+)?|testing\/(mock|snapshot|types))"/g;
```

Comment:
Removed `import \{.*\} ` part as this doesn't work if there are line breaks between `import` and `from`.

Also added `testing\/(mock|snapshot|types)` part as these files also include assertion functions.

Reply:
Having assertions spread across multiple packages is confusing. What if we moved assertions (only) to `@std/assert`? WDYT?

Reply:
That makes `@std/assert` a huge package. In the current layout, assertions are organized by the category (mock / snapshot / types). This makes more sense to me.